web/components/inputs/FcInputLocationSearch.vue

```diff
@@ -9,57 +9,56 @@
       :offset-y="true"
       :open-on-click="false">
       <template v-slot:activator="{ attrs, on: onMenu }">
```

Comment:
same here: use `attrs: attrsMenu`.

we should also check for this same pattern elsewhere - do we have other inputs that need to be wrapped in a `<div>`, to avoid incorrect use of ARIA roles?